src/actions/index.js

```diff
@@ -0,0 +1,48 @@
+import { GET_USERS, DELETE_USER, ADD_USER, SELECT_USER, EDIT_USER } from "../actions/types";
+
+
+export const getUsers = () => {
+    return {
+      type: GET_USERS,
+      payload: ""
+    };
+  };
+
+  export const selectUser = (user) => {
+    return {
+      type: SELECT_USER,
+      payload: user
+    } ;
+  };
+  
+  export const deleteUser = (user) => {
+    try {
+      return {
+        type: DELETE_USER,
+        payload: user
+      };
+    } catch (e) {
+      return {
+        type: DELETE_USER,
+        payload: user
+      } ;
+    }
+  };
+
+  let previousId = 4;
```

Comment:
this previousId seems like a bit of a hack. Is it possible to derive from the state?

Reply:
Sure, I'll update the id based on the last available user object in the list.